### DISCUSSION TITLE
overwrite the maximum values ​​for some RW sensors?

### DISCUSSION THREAD

**Discussion by man55** [kellerza/sunsynk#322] [Q&A] [answered]
How can I overwrite the maximum values ​​for some RW sensors?
For example, for **_Grid Charge Battery current_** and **_Battery Max Charge current_**, since their maximum values ​​are too high for my inverter and this creates inconvenience when setting them with a slider, especially from a smartphone. For example, I can’t get to 50A, the slider jumps and it turns out to be either 49 or 51 )

**Comment by kellerza** [comment, marked as answer]
The only way to do this would be to redefine the sensor as a custom sensor. Have a look at the existing definitions & the custom sensor help page

**Comment by kellerza** [comment]
You can also force numeric value to have a editable box instead of a slider
